pages/dashboard/projects.vue

```diff
@@ -1,22 +1,740 @@
+            title="A page/repository containing the source code for your project"
+          >
+            <span>Source code</span>
+            <input
+              v-model="editLinks.source.val"
+              :disabled="editLinks.source.clear"
+              type="url"
+              maxlength="2048"
+              placeholder="Enter a valid URL"
+            />
+            <button
+              class="square-button label-button"
+              :data-active="editLinks.source.clear"
+              @click="editLinks.source.clear = !editLinks.source.clear"
+            >
+              <TrashIcon />
+            </button>
+          </label>
+          <label
+            title="A page containing information, documentation, and help for the project."
+          >
+            <span>Wiki page</span>
+            <input
+              v-model="editLinks.wiki.val"
+              :disabled="editLinks.wiki.clear"
+              type="url"
+              maxlength="2048"
+              placeholder="Enter a valid URL"
+            />
+            <button
+              class="square-button label-button"
+              :data-active="editLinks.wiki.clear"
+              @click="editLinks.wiki.clear = !editLinks.wiki.clear"
+            >
+              <TrashIcon />
+            </button>
+          </label>
+          <label
+            class="no-margin"
+            title="An invitation link to your Discord server."
+          >
+            <span>Discord invite</span>
+            <input
+              v-model="editLinks.discord.val"
+              :disabled="editLinks.discord.clear"
+              type="url"
+              maxlength="2048"
+              placeholder="Enter a valid URL"
+            />
+            <button
+              class="square-button label-button"
+              :data-active="editLinks.discord.clear"
+              @click="editLinks.discord.clear = !editLinks.discord.clear"
+            >
+              <TrashIcon />
+            </button>
+          </label>
+        </section>
+        <p>Changes will be applied to the following projects:</p>
+        <ul>
+          <li
+            v-for="project in selectedProjects.filter(
+              (it) => (it.permissions & EDIT_DETAILS) !== EDIT_DETAILS
+            )"
+            :key="project.id"
+          >
+            {{ project.title }}
+          </li>
+        </ul>
+        <p>
+          If a project you have selected is not in this list - you do not have
+          permission to edit it's details.
+        </p>
+        <div class="button-group">
+          <button class="iconified-button" @click="$refs.editLinksModal.hide()">
+            <CrossIcon />
+            Cancel
+          </button>
+          <button class="iconified-button" @click="bulkEditLinks()">
+            Apply
+          </button>
+        </div>
+      </div>
+    </Modal>
+    <ModalConfirm
+      ref="deleteBulkModal"
+      :title="`Delete Project${selectedProjects.length > 1 ? 's' : ''}`"
```

Comment:
```suggestion
      :title="`Delete project${selectedProjects.length > 1 ? 's' : ''}`"
```

Sentence case, not title case